src/promptflow/promptflow/_sdk/_service/app.py

```diff
@@ -142,12 +142,14 @@ def monitor_request():
                         "last_request_time"
                     ] > timedelta(hours=PF_SERVICE_HOUR_TIMEOUT):
                         # Todo: check if we have any not complete work? like persist all traces.
-                        app.logger.warning(f"Last http request time: {app.config['last_request_time']} was made "
-                                           f"{PF_SERVICE_HOUR_TIMEOUT}h ago")
+                        app.logger.warning(
```

Comment:
print decent message when we do start_trace()

Reply:
didn't get it. you mean change in the following pr? https://github.com/microsoft/promptflow/pull/2455/files

Reply:
Yes.